.github/workflows/deploy-to-gh-pages.yaml

```diff
@@ -0,0 +1,46 @@
+    name: Deploy to GitHub Pages
+
+    on:
+      push:
+        branches:
+          - main
+      pull_request:
+
+    jobs:
+      deploy:
+        runs-on: ubuntu-24.04
+        permissions:
+          contents: write
+        concurrency:
+          group: ${{ github.workflow }}-${{ github.ref }}
+        steps:
+          - uses: actions/checkout@v3
+
+          - name: Setup Node
+            uses: actions/setup-node@v4
+            with:
+              node-version: "20"
```

Comment:
Devcontainer is using node v22.

The two configurations should match. Both be v20 or v22.

Reply:
Probably? NVM is setup in the image...

I don't really have much experience with NVM.

Reply:
The documentation is here: https://github.com/devcontainers/features/tree/main/src/node

Yes, that is a feature and not an image, but image is just the `base` :heavy_plus_sign: the `node` feature with defaults...